### DISCUSSION TITLE
CocActionAsync('definitionHover'): can I see full definition, without missing fields?

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [neoclide/coc.nvim#4183] [Q&A] [answered]
Here are 22 fields are missed, why?

![image](https://user-images.githubusercontent.com/3514015/189636685-33ab0dff-fba5-4bde-99b7-14d48383d193.png)

I already have this in config: `call coc#config('diagnostic', { 'floatConfig': {'maxHeight': 50 }})`

I think out-of-the-box it should show full definition, even without a config.

**Comment by chemzqm** [comment, marked as answer]
Should be problem of tsserver, checkout https://github.com/neoclide/coc.nvim/wiki/Debug-language-server#using-output-channel
